recipe/meta.yaml

```diff
@@ -20,16 +19,15 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - patch     # [(not win) and (py==311)]
-    - m2-patch  # [win and (py==311)]
+    - patch     # [(not win)]
+    - m2-patch  # [win]
   host:
     - python
     - pip
     - libprotobuf {{ version }}
```

Comment:
Do we need `abseil-cpp`  https://github.com/protocolbuffers/protobuf/blob/e45a6d50682d5d4b204e7d76254b1585b43c6815/src/README.md?plain=1#L22? The conda-forge recipe has `libabseil` in `host` https://github.com/conda-forge/protobuf-feedstock/blob/20df32ca656756cbb2401ec50dcbd70ee2d0fe6b/recipe/meta.yaml#L43

Reply:
it is just name difference, we have abseil-cpp and CF has libabseil, maybe we should another PR for abseil and make alias  for package name as we have in boost also similar situation ?

Reply:
IMO, there is a name difference only in our abseil recipe and cf one.  I think it differs from the boost situation

Reply:
about missing abseil in the host, it is indirect dependency pulled by libprotobuf, eg:

```
The following NEW packages will be INSTALLED:
    ...
    abseil-cpp:       20230802.0-h6a678d5_2  local
    ...
```